composer.json

```diff
@@ -51,6 +51,6 @@
     "bin" : [
         "bin/phiremock"
     ],
-    "minimum-stability" : "dev",
-    "prefer-stable" : true
+    "minimum-stability": "dev",
```

Comment:
What do you think about removing these two stability-related lines? I think they are not needed any more thank to your changes.